solr/core/src/java/org/apache/solr/handler/admin/SolrInfoMBeanHandler.java

```diff
@@ -233,14 +232,14 @@ else if (r != null) {
     }
     return out;
   }
-  
-  @SuppressWarnings({"rawtypes"})
+
+  @SuppressWarnings("unchecked")
   public Object diffObject(Object ref, Object now) {
     if (now instanceof Map) {
-      now = new NamedList((Map)now);
+      now = new NamedList<>((Map<String, ?>)now);
```

Comment:
question: (how) do we know that `<String, ?>` is right here?
```suggestion
      now = new NamedList<>((Map<?, ?>)now);
```

Reply:
Because NamedList won't accept anything else.